definitions/tools/check_strand.cwl

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "how_are_we_stranded_here"
+baseCommand: ['check_strandedness']
+arguments: [
+    "--print_commands"
+]
+
+stdout: "$(inputs.reads1.nameroot).strandness_check.txt"
+
+requirements:
+    - class: ResourceRequirement
+      ramMin: 16000
+      tmpdirMin: 25000
+      coresMin: 4
+    - class: DockerRequirement
+      dockerPull: "smk5g5/checkstranded:latest"
```

Comment:
Is there a specific version we can use? `latest` might change on us and break this pipeline without warning.